schema/spacewalk/common/tables/rhnActionApplyStates.sql

```diff
@@ -27,6 +27,10 @@ CREATE TABLE rhnActionApplyStates
                          DEFAULT ('N') NOT NULL
                          CONSTRAINT rhn_act_apply_states_test_ck
                              CHECK (test in ('Y','N')),
+    direct           CHAR(1)
+                         DEFAULT ('N') NOT NULL
+                         CONSTRAINT rhn_act_apply_states_direct_ck
+                             CHECK (test in ('Y','N')),
```

Comment:
```suggestion
    direct           BOOLEAN NOT NULL DEFAULT FALSE
```

Reply:
```
susemanager=# select pg_column_size(test) as "char(1) size", pg_column_size(direct) as "boolean size" from rhnactionapplystates limit 1;
 char(1) size | boolean size
--------------+--------------
            2 |            1
```

Reply:
Honestly I do not want to have different kind of boolean handling in the same table.
Long term we can think about how to migrate from CHAR(1) to BOOLEAN. But 1 column this and the next the other way sounds scary to me

Reply:
Any maybe this will be replaced anyway with something which specify a executor string. Not sure if we need other executors in the future and a boolean is not enough.